packages/advanced-logic/src/extensions/abstract-extension.ts

```diff
@@ -60,7 +67,12 @@ export abstract class AbstractExtension<TCreationParameters> implements Extensio
         throw Error(`This extension has already been created`);
       }
 
-      copiedExtensionState[extensionAction.id] = this.applyCreation(extensionAction, timestamp);
+      copiedExtensionState[extensionAction.id] = this.applyCreation(extensionAction, timestamp, {
+        extensionsState,
+        extensionAction,
+        requestState,
+        actionSigner,
+      });
```

Comment:
Necessary so we can call `applyActionToExtension` from the meta-pn context to create sub-pn states ([see here](https://github.com/RequestNetwork/requestNetwork/pull/1405/files#diff-37ce446eeafd558b24a48b848941f960cfcf8703c119d6ffc3767318568d12e0R104)).